### ISSUE TITLE
Issues selecting an article

### ISSUE THREAD

**Issue by raman325** [lunik1/docker-tt-rss#10]
I can select one article fine, but when I select the next article, I get the following error and TT-RSS stops functioning:
```
TypeError: Cannot set property 'innerHTML' of null
    at Object.pack (https://rss.mydomain.com/js/Article.js?1619077527:285:50)
    at https://rss.mydomain.com/js/Article.js?1619077527:417:13
    at NodeList.forEach (<anonymous>)
    at Object.setActive (https://rss.mydomain.com/js/Article.js?1619077527:415:48)
    at Object.onLoaded (https://rss.mydomain.com/js/Headlines.js?1619077527:715:13)
    at https://rss.mydomain.com/js/Feeds.js?1619077527:417:16
    at https://rss.mydomain.com/js/common.js?1619077527:215:32
```

This was working fine the other day, so I'm thinking this was introduced when I updated the image from 7d504545af to 8d8559d1d3 yesterday morning

**Comment by lunik1** [comment]
Thanks for the report,

This was already fixed upstream, but the way we were downloading the source meant we only got the first push from upstream on each day. This should be addressed in b6cd179 and I don't have the issue with the latest container build. Can you confirm it's fixed on your end?

**Comment by raman325** [comment]
Confirmed that this was resolved, thanks for the quick fix! I had noticed previously that there was sometimes a gap between the master branch upstream and this image, so glad you were able to look into the root issue and resolve that along the way
